Source/Cake.DotNetTool.Module/DotNetToolPackageInstaller.cs

```diff
@@ -229,7 +238,7 @@ private void RunDotNetTool(PackageReference package, DirectoryPath toolsFolderDi
             {
                 _log.Warning("dotnet exited with {0}", exitCode);
                 var output = string.Join(Environment.NewLine, process.GetStandardError());
-                _log.Verbose(Verbosity.Diagnostic, "Output:\r\n{0}", output);
+                _log.Verbose(Verbosity.Diagnostic, "Output:{0}{1}", Environment.NewLine, output);

```

Comment:
There was another minor bug here. Changed to use `Environment.NewLine` when composing output message.